app_dart/lib/src/model/appengine/branch.dart

```diff
@@ -44,10 +44,10 @@ class Branch extends Model<String> {
 }
 
 class SerializableBranch {
-  const SerializableBranch(this.branch, this.key);
+  const SerializableBranch(this.branch, this.id);
```

Comment:
Do you even need `SerializableBranch`? If `Branch` can be json encoded, why not use that?

Reply:
Ohh this is very true. Thanks a lot for pointing this out!